recipes/scons/all/test_package/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, tools
+from conans.errors import ConanException
+from io import StringIO
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "scons"
+
+    def build(self):
+        scons_path = tools.which("scons")
+        assert scons_path.replace("\\", "/").startswith(self.deps_cpp_info["scons"].rootpath.replace("\\", "/"))
+
+        output = StringIO()
+        self.run("{} --version".format(scons_path), run_environment=True, output=output)
+        text = output.getvalue()
+        if self.deps_cpp_info["scons"].version not in text:
+            raise ConanException("scons --version does not return correct version")
+
+        self.output.info("TMP={}".format(os.environ.get("TMP")))
```

Comment:
I am not sure why do we need to print TMP here, but okay...

Reply:
I have an issue running this on my Windows machine. See https://github.com/conan-io/conan-center-index/pull/814#issuecomment-583888754
At one point, there was some problem with temporary folders.
This is a leftover from debugging.
I'll remove it.